crates/astria-sequencer/src/service/info/mod.rs

```diff
@@ -72,13 +72,12 @@ impl Info {
                     .get_block_height()
                     .await
                     .unwrap_or(0);
-                let app_hash = crate::app_hash::AppHash::from(
-                    self.storage
-                        .latest_snapshot()
-                        .root_hash()
-                        .await
-                        .context("failed to get app hash")?,
-                );
+                let app_hash = self
+                    .storage
+                    .latest_snapshot()
+                    .root_hash()
+                    .await
+                    .context("failed to get app hash")?;
```

Comment:
is calling this an app hash still accurate?

Reply:
yeah, it's returned as the `app_hash` inside the cometbft block